### ISSUE TITLE
"New Project" Disabled

### ISSUE THREAD

**Issue by toward01** [ChurchApps/FreeShow#552]
Hey there! Had Free Show up and running great today. Then all of sudden the project I was working on disappeared - could not pull it up. Then I noticed that the New Project button was not working. I can't add a new a project or work on the one I started. Please Help!

**Comment by vassbo** [comment]
Hi, are you able to locate the app data path (`%AppData%/freeshow` on Windows), and send over the `projects.json` file? I guess something might have gone wrong when saving.

**Comment by toward01** [comment]
I have the app on a Mac mini. I deleted the app then reinstalled it. Still no fix
When the app opens, the project side pops open. Then after one second it closes and disappears. 
I have found the app data folders. Is there one in specific I need to find?
Thank you!


Sent from Yahoo Mail for iPhone


On Thursday, June 6, 2024, 1:32 AM, Kristoffer Vassbø ***@***.***> wrote:




Hi, are you able to locate the app data path (%AppData%/freeshow on Windows), and send over the projects.json file? I guess something might have gone wrong when saving.

**Comment by vassbo** [comment]
Okay. On Mac the AppData should be located here: `~/Library/Application Support/FreeShow/`
Please send over the `projects.json` file, that would be great!

**Comment by toward01** [comment]
Here is the only json file I could find. It's in the Library folder I attached.
Thanks,Tony

 


    On Friday, June 7, 2024 at 12:55:40 AM CDT, Kristoffer Vassbø ***@***.***> wrote:  
 
 


Okay. On Mac the AppData should be located here: ~/Library/Application Support/FreeShow/
Please send over the projects.json file, that would be great!

**Comment by vassbo** [comment]
Thanks. Please send the file to this mail (it does not work via GitHub mail): dev@freeshow.app

**Comment by vassbo** [comment]
Did you by any chance close the left panel? If you have a pink line at the left, you can click on it to open!

**Comment by vassbo** [comment]
@toward01 Did you ever get this fixed? I think I encountered this issue today, and got it fixed for the next version! :) It has been there for 20 months.

When a project and folder was deleted while a project inside that folder was opened, new projects/folders did not show up, as they were placed "inside" of that deleted folder.

**Comment by toward01** [comment]
Yes sir, I believe we did get it fixed. Thanks for checking in on this after so long. We ended up switching over to a PC and had no issues. 
Thanks again!


Sent from Yahoo Mail for iPhone


On Thursday, February 13, 2025, 4:32 AM, Kristoffer Vassbø ***@***.***> wrote:




@toward01 Did you ever get this fixed? I think I encountered this issue today, and got it fixed for the next version! :) It has been there for 20 months.

When a project and folder was deleted while a project inside that folder was opened, new projects/folders did not show up, as they were placed "inside" of that deleted folder.
